app/presenters/signup_presenter.rb

```diff
@@ -35,7 +35,7 @@ def email_filter_by
   end
 
   def can_modify_choices?
-    choices? && choices_formatted.any?
+    choices? && choices_formatted.any? && !(content_item["details"]["email_filter_by"] == "all_selected_facets")
```

Comment:
Is this so we wouldn't need to have `cannot_modify_choices` in the content item?

(The build's failing on linting at the mo)